mix.exs

```diff
@@ -1,13 +1,14 @@
 defmodule Redbird.Mixfile do
   use Mix.Project
-  @version "0.6.0"
+  @version "0.7.0"
```

Comment:
I don't think you need to do this manually. Once all the changes are in, you can run [our release](https://github.com/thoughtbot/redbird/blob/master/bin/release) script, and it'll update this version to the correct one. 

This is how you'd use it: 

```sh
bin/release 0.6.0 0.7.0
```

That'll update the version, create and push a tag to GitHub, and then it'll publish the package to hex.

Reply:
Ah, thanks!